### DISCUSSION TITLE
RU translation

### DISCUSSION THREAD

**Discussion by yhdgms1** [terrarium-earth/Ad-Astra#33] [Q&A] [answered]
Nice mod, thank you for the work! 

I plan to play with friends together with this mod, but it lacks the Russian language. 
I can do the translation of this mod into Russian and maintain it later.

The question: If I make PR - will you accept it?

**Comment by AlexNijjar** [comment, marked as answer]
Yes! That would be great. Please PR into the 1.19-architectury branch as the 1.18 and 1.19 branches will be archived soon.

**Comment by yhdgms1** [comment]
I wrote about "maintain it later", but I don't have the opportunity, unfortunately. 😞 

For some reason the discussion wasn't closed, I'm closing it now.
